docs/site-generator/Makefile

```diff
@@ -0,0 +1,155 @@
+# Default shell
```

Comment:
This is madness!, This shouldn't be a Makefile, this should be a program or a script, you are subverting makefiles here.  I suggest you use sh or bash to rewrite this.